src/components/RangeSlider/RangeSlider.js

```diff
@@ -163,7 +166,7 @@ const RangeSlider = ({
   // KeyDown is called while user presses the arrows and KeyUp is called when the press
   // is done, so the input value will be updated and won't be one step ahead.
   const handleKeyPress = e => {
-    if (e.keyCode === 37 || e.keyCode === 39) {
+    if (e.keyCode === keymap.ARROW_LEFT || e.keyCode === keymap.ARROW_RIGHT) {
```

Comment:
```suggestion
    if ([keymap.ARROW_LEFT, keymap.ARROW_RIGHT].includes(e.keyCode)) {
```
Another way to write it (choose what you prefer - I don't mind)

Reply:
please try to press the keys UP & DOWN or HOME & END or PG UP & PG DN
For my experiments they all moves the "circle" but it does not affect the "tooltip" above